### PR TITLE
Imp: gutenberg cover image alignwide/alignfull compatibility

### PR DESCRIPTION
**Pull request by eri-trabiccolo**
CSS part for #1601

### CODE REVIEW

File: core/class-fire-resources_styles.php [hunk 1]
```diff
@@ -686,7 +686,7 @@ function czr_fn_maybe_write_boxed_layout_inline_css( $_css ) {
                //Add some rules from sm up
                //add padding
                $css_container[]        = sprintf( '@media (min-width: %1$spx){ %2$s{ padding-right: %3$spx; padding-left:  %3$spx; } }',
-                           $css_container_widths[ 'sm' ],
+                           $css_mq_breakpoints[ 'sm' ],
```
Comment (ghost) [inline core/class-fire-resources_styles.php:689]: @eri-trabiccolo  Was that change related to a bug reported for something else ?

Comment (eri-trabiccolo) [reply core/class-fire-resources_styles.php:689]: something I noticed, that was wrong

Comment (eri-trabiccolo) [reply core/class-fire-resources_styles.php:689]: the previous code generated a media query breakpoint too small, as the container width is smaller than the mq_breakpoint value.

Comment (ghost) [reply core/class-fire-resources_styles.php:689]: ok ! thx

**Pull request merged**